Library/Homebrew/extend/os/linux/extend/pathname.rb

```diff
@@ -0,0 +1,12 @@
+class Pathname
+  # @private
+  def elf?
+    read(4) == "\x7fELF"
```

Comment:
This magic warrants a comment or citation.

Reply:
Here's a citation: https://en.wikipedia.org/wiki/List_of_file_signatures

Reply:
Perfect, this would be great in a comment.

Reply:
It might be useful to link to a specific revision: https://en.wikipedia.org/wiki/List_of_file_signatures?oldid=807859331

Reply:
The current ELF document is the better reference in my opinion.